_posts/en/newsletters/2021-03-31-newsletter.md

```diff
@@ -0,0 +1,134 @@
+   via [bolts #780][].
+
+   The paper uses its conclusions to provide a routing algorithm that it
+   claims can reduce payment retry attempts by 20% compared to their
+   simplification of existing routing algorithms.  The new algorithm
+   prefers routes with a higher computed probability of success, whereas
+   existing algorithms use a heuristic approach.  Combined with JIT
+   rebalancing, they estimate a 48% improvement.  Given that each retry
+   usually requires several seconds, and could take much longer in some
+   cases, this could provide a significant benefit to the network.
+   The algorithm was tested against several example networks, including
+   one drawn from a snapshot of almost 1,000 live channels.
+
+   The paper deliberately does not take routing fees into consideration,
+   and most mailing list discussion about the topic focused on how to
+   use the results while still ensuring users don't pay an excessive
+   amount of fees.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 2.0.1][] is a bugfix release that addresses minor issues with Trezor
+  T passphrase entry and keyboard shortcuts in the `hwi-qt` user
+  interface.
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] is a release candidate
+  for the next major version of this LN node software.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #17227][] Qt: Add Android packaging support FIXME:jnewbery
+
+- [Rust-Lightning #849][] makes a channel's `cltv_expiry_delta`
+  configurable and reduces the default value from 72 blocks to 36
+  blocks.  This parameter sets the minimum number of blocks a node will
+  have to settle a payment attempt with its upstream peer after learning
+  from its downstream peer whether or not that payment succeeded; it must
```

Comment:
"This parameter [...] payment succeeded;" is a bit long and difficult to parse. "the minimum number of blocks a node will have to settle" in particular reads like the node has to settle blocks, whereas it really means "the number of blocks within which a node must settle".